### DISCUSSION TITLE
Different Zoom Levels for Left and Right Panel

### DISCUSSION THREAD

**Discussion by 9812ga98sd** [TriliumNext/Trilium#4644] [Q&A]
I have lots of notes an want to use the left panel as an index but when I zoom out, the left panel text is the right size but the right panel text is too small. Can they have independent zooms or locked zooms?

70& zoom for the left panel, 100% zoom for the notes panel.

Thank you.

**Comment by rom1dep** [comment]
I imagine that someone could hack some CSS around this, but an easy way to go with firefox is to open the notes in different tabs while setting `browser.zoom.siteSpecific	false` in about:config.

**Comment by 9812ga98sd** [comment]
I should have mentioned I am using the desktop app.

**Comment by zadam** [comment]
Answered here: https://github.com/zadam/trilium/issues/4645
